VultisigApp/VultisigApp/View Models/JoinKeysignViewModel.swift

```diff
@@ -234,28 +234,37 @@ class JoinKeysignViewModel: ObservableObject {
     }
     
     func blowfishTransactionScan() async throws {
-        guard let payload = keysignPayload else {
-            throw NSError(domain: "JoinKeysignViewModel", code: 1, userInfo: [NSLocalizedDescriptionKey: "Keysign payload is missing."])
-        }
         
-        switch payload.coin.chainType {
-        case .EVM:
-            let blowfishResponse = try await blowfishEVMTransactionScan()
-            blowfishShow = true
-            blowfishWarningsShow = !(blowfishResponse.warnings?.isEmpty ?? true)
-            blowfishWarnings = blowfishResponse.warnings?.compactMap { $0.message } ?? []
-            
-        case .Solana:
-            let blowfishResponse = try await blowfishSolanaTransactionScan()
-            blowfishShow = true
-            blowfishWarningsShow = !(blowfishResponse.aggregated?.warnings?.isEmpty ?? true)
-            blowfishWarnings = blowfishResponse.aggregated?.warnings?.compactMap { $0.message } ?? []
-            
-        default:
-            blowfishShow = false
-            blowfishWarningsShow = false
-            blowfishWarnings = []
-        }
+        // Disabling blowfish until we find a replacement
+        // It is commented since we will reuse the same structure for the next provider
+        
```

Comment:
Please just remove the code , don't comment it out , git has history, you can get it back later

Reply:
sure